book/plugins/general-abilities/js2native/readme.md

```diff
@@ -190,30 +212,34 @@ Please make sure to map any PII data you send to the naming convention of the fi
 * Phone
 * SocialIDs
 
- ### Native player
+ ## Native player
  ### Video.playNative(options)
  Launches native player with specific URL.
  ```javascript
- var options = {
-   video_url: "http://demo.unified-streaming.com/video/tears-of-steel/tears-of-steel.ism/.m3u8"
- };
-  js2n.Video.playNative(options);
+var options = {
+    video_url: "http://demo.unified-streaming.com/video/tears-of-steel/tears-of-steel.ism/.m3u8"
+};
+
+js2n.Video.playNative(options);
 ```
-### Native sharing
+
+## Native sharing
```

Comment:
@snooky23 also here something got wrong with the markdown
![Screen Shot 2019-07-15 at 11 04 03](https://user-images.githubusercontent.com/3192941/61202509-5b8f7180-a6f0-11e9-8926-0af07ce2d77d.png)

Reply:
fixed